src/styles/components/service-table/styles.less

```diff
@@ -44,7 +44,7 @@
     }
 
     .service-status-icon-wrapper {
-      width: 110px;
+      width: 240px;
```

Comment:
Why don't we just get rid of this width style and add `flex-justify-items-space-between` to the parent flex container?

Reply:
love it! will add!